CHANGELOG

```diff
@@ -485,31 +520,14 @@ Miscellaneous:
 
 You can see the full changelog at: https://github.com/py-pdf/PyPDF2/compare/1.26.0...1.27.0
 
-Patch release 1.27.1, 2022-04-08
-
-- Fixed project links on PyPI page after migration from mstamy2
-  to MartinThoma to the py-pdf organization on GitHub
-- Documentation is now at https://pypdf2.readthedocs.io/en/latest/
-
-Patch release 1.27.2, 2022-04-09
-
-- Add Scripts (including `pdfcat`), Resources, Tests, and Sample_Code back to
-  PyPDF2. It was removed by accident in 1.27.0, but might get removed with 2.0.0
-  See https://github.com/py-pdf/PyPDF2/discussions/718 for discussion
+Version 1.25.1, 2015-07-20
```

Comment:
I think that should be below the 1.26.0 one, right?

Reply:
🤦 Yup, it was in the right position before, just needed to update the header for it.

[62b4978](https://github.com/py-pdf/PyPDF2/pull/927/commits/62b49786186e9d6e5de3680b8cc10fd5428d83b7)